pyqmc/reblock.py

```diff
@@ -0,0 +1,55 @@
+import pyblock
+import pandas as pd
+import numpy as np
+from scipy.stats import sem
+
+
+def reblock(data, reblocks, col):
```

Comment:
Not sure I understand why this function is here. pyblock is doing the same thing.

Reply:
I didn't find anything in pyblock that returned the reblocked data, just the statistics about it.

Reply:
Do we need that?